File: comu/com.energyict.comserver.comserver.core.extjs/src/main/web/js/mdc/src/model/StringResponse.js
Ext.define('Mdc.model.StringResponse', {
    extend: 'Ext.data.Model',
    fields: [
        {name: 'response', type: 'string'}
    ],
    proxy: {
        type: 'rest',
        urlTpl: '/api/mds/{resource}',
        reader: {
            type: 'json'
        },
        setUrl: function (resource) {
            this.url = this.urlTpl.replace('{resource}', resource);
        }
    }
});

File: comu/com.energyict.mdc.masterdata.rest/src/main/java/com/energyict/mdc/masterdata/rest/impl/StringResponse.java
package com.energyict.mdc.masterdata.rest.impl;

/**
 * String response wrapper class.
 */
public class StringResponse {

    private final String response;

    StringResponse(String value) {
        this.response = value;
    }

    public String getResponse(){
        return response;
    }
}
